doc/source/serve/tutorials/index.md

```diff
@@ -16,6 +16,7 @@ batch
 web-server-integration
 rllib
 gradio
```

Comment:
remove this page plz

Reply:
Got it, I'll remove the actual file as well.